pages/data-migration/migrate-from-rdbms-directly.md

```diff
@@ -0,0 +1,279 @@
+```
+
+The exercise is to migrate `Bands` and `Musicians` rows as nodes, and the mapping table as
+relationships. The steps below are provided so you can quickly glance at them at any time:
+
+1. Create necessary indices (important for relationship import)
+2. Switch to in-memory analytical mode (to eliminate memory overhead)
+3. Import nodes from corresponding tables
+4. Import relationships from corresponding tables
+5. Switch back to in-memory analytical mode
+
+### 1. Create necessary indices
+In SQL databases, indices are mostly created after the import has finished, as they can slow down 
+importing times. However, in Memgraph it's necesssary to create indices beforehand. Reason for this
+is the relationship import, which needs to find 2 nodes (start node and end node) as its source and
+destination. For node import, if you're doing `CREATE` commands, it's irrelevant. However, if you're
+executing `MERGE` commands, it will still scan over all the nodes, so use corresponding indices in order
+to make the scanning faster.
+
+We will import only the necessary indices:
+```cypher
+CREATE INDEX ON :Musician(id);
+CREATE INDEX ON :Band(id);
+```
+
+### 2. Switch to in-memory analytical mode
+Memgraph currently offers 2 in-memory modes, transactional and analytical. Transactional is often used in
+production deployments. It has all the ACID guarantees and it's safe to use for transactional workloads.
+However, when executing import for the first time and testing, it is beneficial to use in-memory analytical
+mode. 
+
+In-memory analytical mode offers less memory overhead than transactional mode. Also, in-memory analytical mode
+is highly parallelizable when importing nodes and relationships in the way we will see below. However, it does
+not come with write-ahead logging, and it is therefore stripped of the ACID guarantees. More information
+about analytical mode can be found on our [page about storage modes](/fundamentals/storage-memory-usage).
+
+In-memory storage mode is enabled with the following command:
+```cypher
+STORAGE MODE IN_MEMORY_ANALYTICAL;
+```
+
+In the next two sections, we will provide you with different ways of migrating from an external data source.
+One will be migrating the whole table from the RDBMS to Memgraph, and other will be by issuing a query.
+
+### 3. Import nodes from corresponding tables
+The [migrate module](/advanced-algorithms/available-algorithms/migrate) in MAGE has an easy API how to migrate rows from a relational database to Memgraph. We
+can migrate the whole table and create our musician nodes with the following query:
+
+```cypher
+CALL migrate.postgresql('Musicians' , {
+  user:'postgres',
+  password:'xxxxxx',
+  host:'localhost',
+  database:'postgresdb'
+})
+YIELD row
+CREATE (n:Musician {id: row.musician_id, name: row.name, instrument: row.instrument});
+```
+
+We can see that the YIELD clause gives us multiple `row` objects, which are essentially a map which
```

Comment:
Please use the word "iterate" somewhere in here, so the reader understands that all of the rows will be processed one by one.